SignalMessaging/SignalMessaging-Prefix.pch

```diff
@@ -0,0 +1,24 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
```

Comment:
I added a PCH to `SignalMessaging`.  I haven't added a bridging header yet until we have some swift.